Reflection/ReflectionParameter.php

```diff
@@ -0,0 +1,283 @@
+<?php
+
+/**
+ * The <b>ReflectionParameter</b> class retrieves
+ * information about function's or method's parameters.
+ *
+ * @property-read string $name Name of the parameter, same as calling the {@see ReflectionParameter::getName()} method
+ *
+ * @link https://php.net/manual/en/class.reflectionparameter.php
+ */
+class ReflectionParameter implements Reflector
+{
+    /**
+     * Construct
+     *
+     * @link https://php.net/manual/en/reflectionparameter.construct.php
+     * @param callable $function The function to reflect parameters from.
+     * @param string|int $parameter Either an integer specifying the position
+     * of the parameter (starting with zero), or a the parameter name as string.
+     * @throws \ReflectionException if the function or parameter does not exist.
+     */
+    public function __construct(callable $function, $parameter)
+    {
+    }
+
+    /**
+     * Exports
+     *
+     * @link https://php.net/manual/en/reflectionparameter.export.php
+     * @param string $function The function name.
+     * @param string $parameter The parameter name.
+     * @param bool $return Setting to {@see true} will return the export,
+     * as opposed to emitting it. Setting to {@see false} (the default) will do the
+     * opposite.
+     * @return string|null The exported reflection.
+     * @deprecated Since PHP 7.4 and removed in 8.0
```

Comment:
Could you please remove  `Since PHP` words before version and split this into two separate tags
```
@deprecated 7.4
@removed 8.0
```
Otherwise deprecation will not be highlighted properly by PhpStorm

Reply:
`@removed` is a nonexistent tag. 

I mean that this is not in the specification: https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md and implementations: https://docs.phpdoc.org/latest/references/phpdoc/tags/index.html